sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/TypeExaminationOnSystemTypeTest.cs

```diff
@@ -21,6 +21,8 @@
 extern alias csharp;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using csharp::SonarAnalyzer.Rules.CSharp;
+using SonarAnalyzer.UnitTest.TestFramework;
+using SonarAnalyzer.UnitTest.TestFramework;
```

Comment:
After reviewing 300 changed files I'm not sure if I'm seeing double or there are indeed 2 of them 😄